scripts/plot_docking_results.py

```diff
@@ -1,5 +1,5 @@
 import sys, os, argparse
-import numpy as np
+# TODO: Do we need to add plotly to our environment yaml?
```

Comment:
yes! first I think we should figure out the best way to save the env: https://github.com/choderalab/covid-moonshot-ml/issues/41